Common/CultureFix.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Globalization;
+using System.Threading;
+
+namespace Common
+{
+    public static class CultureFix
+    {
+        public static void UseInvariantCulture()
+        {
+            var ic = CultureInfo.InvariantCulture;
+            Thread.CurrentThread.CurrentCulture = ic;
+            Thread.CurrentThread.CurrentUICulture = ic;
+            CultureInfo.DefaultThreadCurrentCulture = ic;
+            CultureInfo.DefaultThreadCurrentUICulture = ic;
```

Comment:
use project setting instead.

netcoreapp3.1:
```
<ItemGroup>
    <RuntimeHostConfigurationOption Include="System.Globalization.Invariant" Value="true" />
</ItemGroup>
```
for netcoreapp5 this will just be:
```
<PropertyGroup>
    <TargetFramework>netcoreapp5</TargetFramework>
    <InvariantGlobalization>true</InvariantGlobalization>
</PropertyGroup>
```